File: Customer/Customer.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package TokenizacionGradle;

import static okhttp3.RequestBody.create;

import java.io.IOException;
import okhttp3.MediaType;
import okhttp3.OkHttpClient;
import okhttp3.Request;
import okhttp3.RequestBody;
import okhttp3.Response;

/**
 *
 * @author alex0360
 */
public class Customer {

    public static String PostCustomer() throws IOException {

        OkHttpClient client = new OkHttpClient()
                .newBuilder()
                .build();
        RequestBody body = create(MediaType.parse("application/json; charset=utf-8"), "{\n"
                + "\"Email\": \"edelossantos@cardnet.com.do\",\n"
                + "\"ShippingAddress\": \"Calle Max Henriquez Ureña No. 6\",\n"
                + "\"BillingAddress\": \"Calle 1 No 10, Piantini\",\n"
                + "\"FirstName\": \"Emmanuel\",\n"
                + "\"LastName\": \"De los Santos\",\n"
                + "\"DocumentTypeId\": 2,\r\n"
                + "\"DocNumber\": \"00114918123\",\n"
                + "\"PhoneNumber\": \"8096042082\" ,\n"
                + "\"Enable\": true"
                + "}");
        Request request = new Request.Builder()
                .url("https://lab.cardnet.com.do/servicios/tokens/v1/api/Customer")
                .method("POST", body)
                .addHeader("Content-Type", "application/json")
                .addHeader("Authorization", "Basic 9kYH2uY5zoTD-WBMEoc0KNRQYrC7crPRJ7zPegg3suXguw_8L-rZDQ__")
                .build();
        try (Response response = client.newCall(request).execute()) {
            return response.body().string();
        }
    }

    public static String GetCustomer() throws IOException {

        OkHttpClient client = new OkHttpClient()
                .newBuilder()
                .build();
        Request request = new Request.Builder()
                .url("https://lab.cardnet.com.do/servicios/tokens/v1/api/Customer/1047")
                .method("GET", null)
                .addHeader("Content-Type", "application/json")
                .addHeader("Authorization", "Basic 9kYH2uY5zoTD-WBMEoc0KNRQYrC7crPRJ7zPegg3suXguw_8L-rZDQ__")
                .build();

        try (Response response = client.newCall(request).execute()) {
            return response.body().string();
        }
    }
    
    public static String UpdateCustomer() throws IOException {

        OkHttpClient client = new OkHttpClient()
                .newBuilder()
                .build();
        RequestBody body = RequestBody.create(MediaType.parse("application/json; charset=utf-8"), "{\n"
                + "\"Email\": \"edelossantos@cardnet.com.do\",\n"
                + "\"ShippingAddress\": \"Calle Max Henriquez Ureña No. 6\",\n"
                + "\"BillingAddress\": \"Calle 1 No 10, Piantini\",\n"
                + "\"FirstName\": \"Emmanuel\",\n"
                + "\"LastName\": \"De los Santos\",\n"
                + "\"DocumentTypeId\": 2,\r\n"
                + "\"DocNumber\": \"00114918123\",\n"
                + "\"PhoneNumber\": \"8096042082\" ,\n"
                + "\"Enable\": true"
                + "}");
        Request request = new Request.Builder()
                .url("https://lab.cardnet.com.do/servicios/tokens/v1/api/Customer/1047/update")
                .method("POST", body)
                .addHeader("Content-Type", "application/json")
                .addHeader("Authorization", "Basic 9kYH2uY5zoTD-WBMEoc0KNRQYrC7crPRJ7zPegg3suXguw_8L-rZDQ__")
                .build();
        try (Response response = client.newCall(request).execute()) {
            return response.body().string();
        }
    }

}

File: Customer/Customer.py
import requests

def GetCustomer():
    url = "https://lab.cardnet.com.do/servicios/tokens/v1/api/Customer/100"

    headers = {
        'Content-Type': 'application/json',
        'Authorization': 'Basic 9kYH2uY5zoTD-WBMEoc0KNRQYrC7crPRJ7zPegg3suXguw_8L-rZDQ__'
    }

    response = requests.get(url, headers=headers)

    print(response.json())

def PostCustomer():
    url = "https://lab.cardnet.com.do/servicios/tokens/v1/api/Customer/"
    headers = {
        'Content-Type': 'application/json',
        'Authorization': 'Basic 9kYH2uY5zoTD-WBMEoc0KNRQYrC7crPRJ7zPegg3suXguw_8L-rZDQ__'
    }

    params = {"Email": "edelossantos24@cardnet.com.do",
            "ShippingAddress": "Calle Max Henriquez Ureña No. 6",
            "BillingAddress": "Calle 1 No 10, Piantini",
            "FirstName": "Emmanuel",
            "LastName": "De los Santos",
            "DocumentTypeId": 2,
            "DocNumber": "00114918123",
            "PhoneNumber": "8096042082",
            "Enable": "True"}

    response = requests.post(url, headers=headers, json=params)
    print(response.json())

def UpdateCustomer():
    url = "https://lab.cardnet.com.do/servicios/tokens/v1/api/Customer/1047/update"
    headers = {
        'Content-Type': 'application/json',
        'Authorization': 'Basic 9kYH2uY5zoTD-WBMEoc0KNRQYrC7crPRJ7zPegg3suXguw_8L-rZDQ__'
    }

    params = {"CustomerID": 1047,
            "Email": "edelossantos@cardnet.com.do",
            "ShippingAddress": "Calle Max Henriquez Ureña No. 6",
            "BillingAddress": "Calle 1 No 10, Piantini",
            "FirstName": "Emmanuel",
            "LastName": "De los Santos",
            "DocumentTypeId": 2,
            "DocNumber": "00114918123",
            "PhoneNumber": "8096042082",
            "Enable": "True"}

    response = requests.post(url, headers=headers, json=params)
    print(response.json())